source/MRMesh/MRCNCMachineSettings.h

```diff
@@ -17,12 +18,17 @@ struct CNCMachineSettings
         C,
         Count
     };
-    using RotationAxisOrder = std::vector<RotationAxisName>;
+    using RotationAxesOrder = std::vector<RotationAxisName>;
 
+    MRMESH_API void setRotationParams( RotationAxisName paramName, const Vector3f& rotationAxis );
+    MRMESH_API Vector3f getRotationParams( RotationAxisName paramName ) const;
```

Comment:
mb rename this function? `params` looks not representing

Reply:
fixed